upcloud/request/load_balancer_helpers.go

```diff
@@ -31,6 +31,15 @@ func NewLoadBalancerHTTPRedirectAction(location string) upcloud.LoadBalancerActi
 	}
 }
 
+func NewLoadBalancerHTTPRedirectSchemeAction(scheme upcloud.LoadBalancerActionHTTPRedirectScheme) upcloud.LoadBalancerAction {
```

Comment:
In addition to adding this, maybe the `NewLoadBalancerHTTPRedirectAction` could be renamed to `NewLoadBalancerHTTPRedirectLocationAction` for clarity. Would had to be done in a next major release tho, as it's a breaking change.